test/libsolidity/semanticTests/viaYul/array_storage_push_empty.sol

```diff
@@ -13,11 +13,11 @@ contract C {
 // compileViaYul: also
 // ----
 // pushEmpty(uint256): 128
-// gas irOptimized: 635248
+// gas irOptimized: 629616
 // gas legacy: 607287
 // gas legacyOptimized: 589048
 // pushEmpty(uint256): 256
-// gas irOptimized: 870384
+// gas irOptimized: 859120
```

Comment:
Quite a reduction in this file :)

Reply:
I think it allows certain functions to be inlined into a function that used to be just above the size limit. I can create a diff to see.

Reply:
Another case was inlined too, so likely that is the cause.

Reply:
Yes, it inlines the `i++`, nothing else and no subsequent optimizations from the inlining. The loop has 256 iterations, though.